src/reanimated2/js-reanimated/index.ts

```diff
@@ -47,14 +48,18 @@ export const _updatePropsJS = (
   }
 };
 
-const setNativeProps = (
+const updatePropsDOM = (
   component: JSReanimatedComponent,
   style: StyleProps
 ): void => {
   const previousStyle = component.previousStyle ? component.previousStyle : {};
   const currentStyle = { ...previousStyle, ...style };
   component.previousStyle = currentStyle;
-  component.setNativeProps({ style: currentStyle });
+
+  const domStyle = createReactDOMStyle(currentStyle);
+  for (const key in domStyle) {
```

Comment:
We should handle this situation specifically because `react-native-web` has already removed array transform. You can check it out [here](https://github.com/necolas/react-native-web/blob/39b94b194594adebdfd5d4a5b873ced8d387734e/packages/react-native-web/src/exports/StyleSheet/preprocess.js#L203).

Something just like this: 

```
import { createTransformValue } from 'react-native-web/dist/exports/StyleSheet/preprocess';
for (const key in domStyle) {
  if(key === 'transform') {
      (component.style).transform = createTransformValue(domStyle[key]);
  }else{
      (component.style)[key] = domStyle[key];
  }
}

```

Reply:
does the current RNW function not account for it?

Reply:
Uunfortunately no. If you don't use `createTransformValue`, `useAnimatedStyle` will not work.